docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -32,11 +32,7 @@ RUN echo torch=$VERSION
 # TODO: We might need to specify proper versions that work with a specific torch version (especially for past CI).
 RUN [ "$PYTORCH" != "pre" ] && python3 -m pip install --no-cache-dir -U $VERSION torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/$CUDA || python3 -m pip install --no-cache-dir -U --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/$CUDA
 
-RUN python3 -m pip install --no-cache-dir -U tensorflow==2.12
-# (temporarily) The above TF installation upgrade protobuf to v4, which we haven't tested yet!
-# (Before the above line, the installed TF is 2.11 due to some conflict)
-RUN python3 -m pip install --no-cache-dir -U protobuf==3.20.3
-RUN python3 -m pip install --no-cache-dir -U tensorflow_probability
+RUN python3 -m pip install --no-cache-dir -U tensorflow==2.12 protobuf==3.20.3 tensorflow_text tensorflow_probability
```

Comment:
Just for my understanding - why does installing tensorflow_text here keep the 2.12 TF version?

Reply:
The change here in this PR is to upgrade the `tensorflow_text`. Without this line, the currently install version will be `tensorflow_text==2.11.x`. And while we have TF 2.12 (even without this PR), this causes problem when running CI.
The dependency is kind a bit mess, it depends on how we install things, like .[dev] or .[tf-cpu] etc.

Reply:
And with this PR, the `tensorflow_text` will be `2.12.1` which is compatible with TF 2.12, so TF version won't be changed by this line.

Reply:
Gotcha, thanks for explaining!